sway/tree/workspace.c

```diff
@@ -819,7 +819,7 @@ size_t workspace_num_tiling_views(struct sway_workspace *ws) {
 }
 
 static void count_sticky_containers(struct sway_container *con, void *data) {
-	if (container_is_floating(con) && con->is_sticky) {
```

Comment:
This was using `container_is_floating` rather than `container_is_floating_or_child`, but the overall result should remain the same. (The result of `workspace_num_sticky_containers` is currently only compared against 0.)

Reply:
Although this is only used to check for the existence of sticky containers and wouldn't cause any issues, it does make it so the function name(s) are misleading.

`if (container_is_sticky(con) && !con->parent)` would be a logical equivalent and would allow for the naming to remain the same.

Reply:
I've introduced an `container_is_sticky_or_child` variant to keep the semantics of existing code more clear.